src/Module.Server/Common/Models/CrpgAgentApplyDamageModel.cs

```diff
@@ -125,6 +125,11 @@ public CrpgAgentApplyDamageModel(CrpgConstants constants)
             finalDamage *= 0.23f; // Decrease damage from couched lance.
         }
 
+        if (attackInformation.DoesAttackerHaveMountAgent && attackInformation.IsAttackerAgentDoingPassiveAttack)
+        {
+            finalDamage *= 0.23f; // Decrease damage from couched lance.
+        }
+
```

Comment:
There's a change this does not work. Please add a debug print next to finaldamage to check that this line is actualy being exectured and report back

Reply:
I don't actually recall ever adding this in place so I actually have no idea where these lines came from, I've certainly never touched anything relating to couch lance to my memory so idk wtf happened there, will look into what commit put this in but i will probably discard because i did not intend to make any changes to this.

Reply:
found it, i have actually no idea how i did this:
https://github.com/namidaka/crpg/pull/287/commits/d7cd170caefe3730990edc80ad1ae81da4155941

i'll discard this change as well as it is not intended at all, apologies.